manual-source/src/lib.rs

```diff
@@ -17,45 +17,54 @@ use std::{sync::Arc, time::Duration, usize};
 use zenoh_flow::{bail, prelude::*};
 use zenoh_flow_example_types::ZFUsize;
 
-struct ManualSource;
+struct ManualSource {
+    output: Output,
+}
+
+#[async_trait]
+impl Node for ManualSource {
+    async fn iteration(&self) -> Result<()> {
+        println!("> Please input a number: ");
+        let mut number = String::new();
+        async_std::io::stdin()
+            .read_line(&mut number)
+            .await
+            .expect("Could not read number.");
```

Comment:
This comment is not directed to your PR, but I was thinking that maybe the manual source should read from a file (the file name can be configured, and the default one can be `/tmp/manual-source.in`.

So we can imagine that once this flow is instantiated if we write something on a file: `echo "value" > /tmp/manual-source.in` then we are actually having a kind of manual source, without requiring the daemon to have access to a TTY.

Otherwise, when running Zenoh-Flow as a daemon there is no way this source can work.
@J-Loudet opinions?

Reply:
As far as I recall, this source was created when the Daemon was not yet a reality.
Updating it as you suggest seems like a good (if not the best) approach!